TractorMod/Framework/Attachments/PickaxeAttachment.cs

```diff
@@ -77,15 +85,15 @@ public override bool Apply(Vector2 tile, SObject tileObj, TerrainFeature tileFea
 
                 // clear dead crops
                 if (dirt.crop?.dead == true)
-                    return this.UseToolOnTile(tool, tile);
+                    return this.ClearDeadCrops && this.UseToolOnTile(tool, tile);
             }
 
             // clear boulders / meteorites
             // This needs to check if the axe upgrade level is high enough first, to avoid spamming
             // 'need to upgrade your tool' messages. Based on ResourceClump.performToolAction.
             {
                 ResourceClump clump = this.GetResourceClumpCoveringTile(location, tile);
-                if (clump != null && this.ResourceUpgradeLevelsNeeded.ContainsKey(clump.parentSheetIndex) && tool.upgradeLevel >= this.ResourceUpgradeLevelsNeeded[clump.parentSheetIndex])
+                if (clump != null && this.ResourceUpgradeLevelsNeeded.ContainsKey(clump.parentSheetIndex) && tool.upgradeLevel >= this.ResourceUpgradeLevelsNeeded[clump.parentSheetIndex] && this.ClearDebris)
```

Comment:
A separate config field for clearing boulders / meteorites might be useful here.

Reply:
You think one or two settings here, I'm thinking one but I can do either.